src/main/java/net/minestom/server/thread/ThreadDispatcher.java

```diff
@@ -236,7 +239,9 @@ private void processLoadedPartition(P partition) {
         this.partitions.put(partition, partitionEntry);
         this.partitionUpdateQueue.add(partition);
         if (partition instanceof Tickable tickable) {
-            processUpdatedElement(tickable, partition);
+            if (!(tickable instanceof Instance)) { // Instances are ticked separately by server process
```

Comment:
This is a bit tricky - I generally think that the behavior of ticking instances separately at the start of the tick doesn't make sense for a per-instance scheme. Though at the same time I would expect instances to be ticked before chunks and entities so some kind of ordering would be required here. Not sure the best solution.